base_delivery_carrier_label/models/stock_move_line.py

```diff
@@ -36,10 +36,9 @@ def get_weight(self):
                     "Type conversion not implemented for product %s", product.id
                 )
                 cant_calc_total = True
-            # reserved_qty may be 0 if you don't set move line
+            # quantity_product_uom may be 0 if you don't set move line
             # individually but directly validate the picking
-            qty = operation.qty_done or operation.reserved_qty
```

Comment:
`qty = operation.quantity or operation.quantity_product_uom`

From 16.0 to 17.0, in `stock.move.line`:

- `reserved_qty` is removed. `quantity_product_uom` seems to replace it.
- `qty_done` is removed. `quantity` seems to replace it.